### PR TITLE
created: button html & style

### PR DESCRIPTION
**Pull request by Gautam2305**
- made 6 buttons of different kinds.
- two solid buttons: one primary CTA & another for add to cart
- one secondary CTA button with outline.
- link button which changes color on hover
- cart icon button
- wishlist icon button
- a floating action button
Screenshot (78)

### CODE REVIEW

File: button.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <a class="btn btn-outline-primary" href="#"> Primary-Outline </a>
+    <a class="link-text link-text-primary" href="#"> Primary-Link </a>
+    <a class="btn btn-icon-primary" href="#"><span><i class="fas fa-shopping-cart"></i></span></a>
+    <a class="btn btn-icon-primary" href="#"><span><i class="far fa-heart"></i></span></a>
+    <a class="btn btn-icon-text-primary" href="#"><span class="btn-icon-text"><i class="fas fa-shopping-cart"></i> Add to cart </span></a>
+    </div>
+    <div class="floating-action-button">
+    <a class="btn-floating" href="#"> + </a>
+    </div>
+
```
Comment (nishasen) [inline button.html]: extra space here

File: style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @import "stylesheets/Components/alert.css";
-
+@import "stylesheets/Components/avatar.css";
+@import "stylesheets/Components/button.css";
```
Comment (nishasen) [inline style.css]: Mention the component elements below the CSS, so that other general files do not overshadow your code

**Review by nishasen — CHANGES_REQUESTED** [review]

File: button.html (hunk 1 above)
Comment (Gautam2305) [reply button.html]: resolved.

File: style.css (hunk 2 above)
Comment (Gautam2305) [reply style.css]: done . do tell if anything else is required to be done

File: stylesheets/CSS/typography.css [hunk 3]
```diff
@@ -48,4 +48,14 @@ p {
...
+.btn{
+    font-family: var(--st-special-text);
+    font-size: 1.25rem;
+    font-weight: 600;
+}
+.link-text{
+    font-family: var(--st-body-text);
+    font-size: 1rem;
+    font-weight: 500;
 }
```
Comment (shuntzhere) [inline stylesheets/CSS/typography.css:61]: If you are using multiple font families, you can directly use their names as that would make more sense.

**Review by shuntzhere — COMMENTED** [review]
The PR looks good. Suggested minor variable changes.

**Review by nishasen — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (nishasen) [reply style.css]: Looks good

**Pull request merged**